### PR TITLE
Remove noise from git changes on Windows

### PR DESCRIPTION
**Pull request by k-rus**
Fixes null device on Windows to work in both CMD and PowerShell.
Ignores scratch files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,5 @@ typedef.list
 
 /CMakeSettings.json
 /out/*
+
+scratch_file*
```
Comment (svenklemm) [inline .gitignore:29]: you sure this is right path, arent those created in the sql folder

Comment (k-rus) [reply .gitignore:29]: You are right that they are created in `sql` folder. However, I guess the same usage of scratch files might be implemented in other places in future, thus I suggest to ignore them everywhere.

Comment (k-rus) [reply .gitignore:29]: This notation is enough for `gitignore` to ignore the files in any folder.

**Comment by k-rus** [comment]
> i think scratch_file should be cleaned up instead of being ignored

cc @gayyappan as the committer of the original change.

**Pull request closed without merging**